TitoAgudelo/enterpriseNT
# USER-1 Profiles

# overview
CRUD PUT profile detail, utilizábamos el método de test 

# test
- git checkout profiles
- git pull origin profiles
- run gulp serve-dev in terminal
- tab profile select a detail of any profile
- update the detail in the form and submit
- expect detail profile updated
